src/components/Icon/index.js

```diff
@@ -25,6 +25,9 @@ const propTypes = {
 
     /** Is inline icon */
     inline: PropTypes.bool,
+
+    /** Is svg avatar icon */
+    svgAvatar: PropTypes.bool,
```

Comment:
I think we prefer using boolean variables that start with `is` or `has` right? I feel like this could be a better prop name:

```suggestion
    /** Is SVG avatar icon */
    isSvgAvatar: PropTypes.bool,
```

Reply:
Also i'm now wondering: Since we render lots of SVGs using this Icon component, what's special about avatar SVGs that requires a special avatar SVG border? It would be nice (maybe in a follow up PR) if we can get rid of this so all SVGs are treated equally, right? Iffff possible 🙏

Reply:
That's a good point! The reason for this was that I couldn't get the rounded border styling of avatars to apply without applying it directly to the Icon component (all default Avatar SVGs are square). There's probably another better way to do it though